rules_php_gapic/php_gapic.bzl

```diff
@@ -61,15 +61,17 @@ def php_gapic_srcjar(
     if grpc_service_config:
         plugin_file_args[grpc_service_config] = "grpc_service_config"
 
+    # Generate the gapic_metadata.json file.
+    plugin_args = ["metadata"]
+
+    # Transport.
     if transport == None:
         transport = "grpc+rest"
     if transport == "grpc":
         fail("Error: gRPC-only PHP GAPIC libraries are not yet supported")
     if transport != "grpc+rest" and transport != "rest":
         fail("Error: Only 'grpc+rest' or 'rest' transports are supported")
-
-    # Generate the gapic_metadata.json file.
-    plugin_args = ["metadata"]
+    plugin_args.append("transport=%s" % transport)
```

Comment:
Why was this moved? It is not not a big deal, but it seems like the `transport` ifs block does not have impact on this value, so it is usually better to keep modifications to the same variable (`plugin_args` in this case) together (line 65 and line 74 in this case).

Reply:
The move allows us to group the `transport` setting logic together more tightly. Either has its merits, so I've moved it back.